.github/dependabot.yml

```diff
@@ -16,3 +16,8 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "terraform"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Since you don't have any Terraform in your root directory, you can get rid of this block:
```suggestion
```

Reply:
BTW @zenine07, I forgot to check that in the other repo I reviewed for you.  Can you please double-check that?

Reply:
@dav3r Made the change here, I can go make the change in the other repo. It was in it still

Reply:
Thanks!